raiden/blockchain_events_handler.py

```diff
@@ -111,6 +120,7 @@ def handle_channel_new(raiden: "RaidenService", event: Event):
         )
         raiden.handle_and_track_state_change(new_channel)
 
+        # pylint: disable=E1101
         partner_address = channel_state.partner_state.address
```

Comment:
Removing `field(repr=False)` in the case where we only disable repr for these fields is possible. However, there will be many different places where we need to override the field with something else.

Reply:
Sure, but this would fix roughly 32 instances that are just `repr=True`. Counted with:

> ag --python 'field\(repr=False\)' --nobreak --nofilename | wc

Reply:
I checked this point out. Most of the `disable=E1101` were added because `partner_state` and `our_state` are defined as `field(repr=False)`. However, i would argue that setting them to be included in `repr` would be a bit too much because of the amount of data that will end up being logged.
Do you think we should accept this and remove `field` assignment to those attributes?